.github/workflows/linux/build/action.yml

```diff
@@ -11,6 +11,8 @@ runs:
       uses: "shivammathur/setup-php@v2"
       with:
         php-version: "${{ inputs.version }}"
+        # Only install required extensions
+        extensions: "none,date,json,spl,standard,xml"
```

Comment:
What is "none"?

Reply:
`none` indicates that no default extensions should be installed. The action allows disabling individual extensions using `:<extension>` or disabling all using `none` followed by specifying extensions we want. I figured that was more sensible than only excluding the mongodb extension. This will also point out if we introduce a dependency on a new extension.